SharpBucketTests/V2/EndPoints/FilterBuilderTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SharpBucket.V2.EndPoints;
+using Shouldly;
+
+namespace SharpBucketTests.V2.EndPoints
+{
+    [TestFixture]
+    public class FilterBuilderTests
+    {
+        [TestCase("foo=bar", "foo=bar")]
+        [TestCase("foo='bar'", "foo=\"bar\"")]
+        [TestCase("foo='bob''s burgers'", "foo=\"bob's burgers\"")]
+        [TestCase("", "")]
+        [TestCase(null, null)]
+        public void Foo(string input, string expected)
```

Comment:
To respect the existing code style, name the method like the tested one followed by an underscore and a test context description
```suggestion
        public void ParseSingleQuotedString_ForGivenInputAndExpected(string input, string expected)
```

Reply:
Sorry -- I didn't mean to leave Foo in there!